cost/kubecost/cluster/kubecost_cluster_rightsizing_recommendations.pt

```diff
@@ -162,79 +162,114 @@ script "js_target_util", type:"javascript" do
   code "result = { value: (param_target_util / 100).toString() }"
 end
 
-datasource "ds_cluster_sizing" do
+datasource "ds_clusters" do
   request do
     host $param_kubecost_host
-    path "/model/savings/clusterSizing"
+    path "/model/savings/clusterSizingETL"
     query "minNodeCount", $param_min_nodes
     query "window", join([$param_lookback, "d"])
     query "targetUtilization", val($ds_target_util, "value")
+    query "allowSharedCore", "true"
     header "User-Agent", "RS Policies"
   end
   result do
     encoding "json"
-    field "accountID", jmes_path(response, "data.parameters.clusterId")
-    field "accountName", jmes_path(response, "data.parameters.clusterName")
-    field "totalNodeCount", jmes_path(response, "data.currentClusterInfo.totalCounts.totalNodeCount")
-    field "totalRAMGB", jmes_path(response, "data.currentClusterInfo.totalCounts.totalRAMGB")
-    field "totalVCPUs", jmes_path(response, "data.currentClusterInfo.totalCounts.totalVCPUs")
-    field "monthlyRate", jmes_path(response, "data.currentClusterInfo.monthlyRate")
-    field "recommendations", jmes_path(response, "data.recommendations")
+    field "accountID", jmes_path(response, "data")
   end
 end
 
+datasource "ds_get_cluster_list" do
```

Comment:
Where is this datasource used? If not used, then it should be removed from the template along with the `js_get_cluster_list` script

Reply:
done